### ISSUE TITLE
Does is support Bangla [BN] language? 

### ISSUE THREAD

**Issue by mahmudtopu3** [typesense/typesense#931]
I tried to search the word "লারাভেল" but it doesn't return anything. 
My data has the word.

**Comment by kishorenc** [comment]
Yes it should work. Please email a sample dataset and query to contact@typesense.org and we will be happy to go through what's happening. Thanks.
